### PR TITLE
[67] Source compatibility Test Suite

### PR DESCRIPTION
**Pull request by BasThomas**

### CODE REVIEW

File: _drafts/2017-04-27-issue-67.md [hunk 1]
```diff
@@ -24,7 +24,9 @@ This week, JP Simard and Jesse Squires take a closer look at [SE-0166](https://g
 
 ### News and community
 
-> TODO
+Apple [announced](https://swift.org/blog/swift-source-compatibility-test-suite/) a test suite for Swift source compatibility, where you can add *your* open source project to the test suite! 
```
Comment (jpsim) [inline _drafts/2017-04-27-issue-67.md]: U+F631 is "deliberately not assigned to any character": https://codepoints.net/U+f631

What character did you mean to insert here?

Comment (BasThomas) [inline _drafts/2017-04-27-issue-67.md]: Ugh, having issues with emoji on iPad+GitHub for no apparent reason. Think it was a "😱"

Comment (BasThomas) [inline _drafts/2017-04-27-issue-67.md]: Thanks for catching that!

Comment (jessesquires) [inline _drafts/2017-04-27-issue-67.md]: lol @jpsim how do you know that!?

Comment (jpsim) [inline _drafts/2017-04-27-issue-67.md]: Safari doesn't render it but chrome renders it as a blank square.  So I pulled it down and checked the code point value.  Then I looked it up.

Comment (BasThomas) [inline _drafts/2017-04-27-issue-67.md]: So the strange thing is that Safari does actually render them if I insert an emoji in the editor, but not afterwards, when you view the source or the parsed MD. 🤷‍♂️ Probably it's a bug on GitHub's end?
img_0284

**Pull request merged**